internal/pkg/cli/dialog/inputs.go

```diff
@@ -0,0 +1,151 @@
+package dialog
```

Comment:
V tomto PR je iba popresuvany kod:

Pred:
![image](https://user-images.githubusercontent.com/19371734/154048099-eb13faf0-c8b0-4cb4-a2b6-74caaaeac091.png)

Po:
![image](https://user-images.githubusercontent.com/19371734/154047986-540ea724-18f9-45a3-916d-614ad28f401c.png)

Reply:
`inputs_select` obsahuje existujuci dialog, kde sa vybera, ktore fields maju byt nahradene za user input.

a `inputs_detail` bude obsahovat novy dialog, kde sa kazdemu `input` nastavi `name` a `description`.

Reply:
Ak by som to nechal v jednom subore, tak by tam bolo prilis vela riadkov.